User Dashboard Page: donations grid #18

Nia-pb opened the pull request:
added the Recent Donations table, other tables still need to be done.

Nia-pb committed:
Added recentDonations.jsx

Nia-pb committed:
RecentDonations Table

Connor-Skudlarek committed:
Merge branch 'main' into donations-grid

Connor-Skudlarek commented on components/UserDashboard/UserDashboard.jsx:
```diff
@@ -0,0 +1,10 @@
+
+import RecentDonations from "./RecentDonations";
+
+export default function UserDashboard() {
```
Should change this to Page() and move to /app/dashboard/page.jsx

I'll handle it so we can get it pushed out. 👍

Connor-Skudlarek commented on components/UserDashboard/RecentDonations.jsx:
```diff
@@ -0,0 +1,40 @@
+import { useEffect, useState } from "react";
+import DummyData from "../../utils/DummyData.js";
+
+export default function ReentDonations() {
```
Typo. I'll get it.

Connor-Skudlarek reviewed (changes requested):
Not ready for PR. Still a lot of tasks to complete.

Connor-Skudlarek committed:
Remove UserDashboard and correct typo

The UserDashboard() component is really acting like the Page() component. The structure of the dashboard should be built up by the components in the UserDashboard component folder rather than taking a pre-built one if at all possible.

Connor-Skudlarek committed:
Merge branch 'main' into donations-grid

Connor-Skudlarek committed:
Merge branch 'main' into donations-grid

Connor-Skudlarek committed:
Move and add to /user/dashboard, add some display CSS

Connor-Skudlarek committed:
Add pagination

Connor-Skudlarek committed:
Add local dummy data for order history

Connor-Skudlarek committed:
Merge branch 'main' into donations-grid

Connor-Skudlarek commented:
Something wrong with the github action check. Currently runs build without error.
